### ISSUE TITLE
SpaceBeforeBrace doesn't lint single-line blocks

### ISSUE THREAD

**Issue by chris-dura** [brigade/scss-lint#94]
If I'm correct in understanding the `SpaceBeforeBrace` linter, this line should throw an error because there's no space before the opening brace?

``` scss
.single-line-selector{color: red;}
```

The only error I get when I lint that line is that `red` should be written as `#f00`.
